src/platforms/hosted/dap.h

```diff
@@ -66,6 +66,9 @@ typedef enum dap_led_type {
 
 #define DAP_QUIRK_NO_JTAG_MUTLI_TAP (1U << 0U)
 
+#define DAP_CMD_BLOCK_WRITE_HDR_LEN 5U // Size of block request write header
```

Comment:
Rather than introducing these in the dap.h header, given they're internal to the DAP implementation would it not be better to put them in dap_command.h? That also prevents the cross-linking of the headers

Reply:
That does seem like a much better idea, can define them using `offsetof()` as well and remove the static asserts!

I think I got confused here about the layering (which layer is meant to depend on which). Will change.

Reply:
Done!